src/elements/element.line.js

```diff
@@ -44,50 +42,51 @@
 
 			// Draw the background first (so the border is always on top)
 			helpers.each(this._children, function(point, index) {
+
 				var previous = helpers.previousItem(this._children, index);
 				var next = helpers.nextItem(this._children, index);
 
-				// First point only
-				if (index === 0) {
-					ctx.moveTo(point._view.x, point._view.y);
-					return;
+				// First point moves to it's starting position no matter what
+				if (!index) {
+					ctx.moveTo(point._view.x, vm.scaleZero);
 				}
 
-				// Start Skip and drag along scale baseline
-				if (point._view.skip && vm.skipNull && !this._loop) {
-					ctx.lineTo(previous._view.x, point._view.y);
-					ctx.moveTo(next._view.x, point._view.y);
+				// Skip this point, draw to scaleZero, move to next point, and draw to next point
+				if (point._view.skip && !this.loop) {
+					ctx.lineTo(previous._view.x, vm.scaleZero);
+					ctx.moveTo(next._view.x, vm.scaleZero);
+					return;
 				}
-				// End Skip Stright line from the base line
-				else if (previous._view.skip && vm.skipNull && !this._loop) {
```

Comment:
Yes I believe so for now. Both features were broken to some extent. I think it's better we go with a solid base. If someone down the line needs to draw falsey values as zeros I think they would probably just scrub their data for that and then expect chartjs to deliver accurately.